packages/eslint-plugin/src/configs/react.js

```diff
@@ -216,6 +143,93 @@ const config = {
     'operator-assignment': 'off',
     'prefer-destructuring': 'off',
     'prefer-template': 'off',
+    // airbnb or other config overrides (some temporary)
+    // TODO: determine which rules we want to enable, and make needed changes (separate PR)
+    'arrow-body-style': 'off',
+    'class-methods-use-this': 'off',
+    'consistent-return': 'off',
+    'default-case': 'off',
+    'func-names': 'off',
+    'global-require': 'off',
+    'spaced-comment': 'off',
+    // airbnb options ban for-of which is unnecessary for TS and modern node (https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L334)
+    // but this is a very powerful rule we may want to use in other ways
+    'no-restricted-syntax': 'off',
+    // permanently disable because we disagree with these rules
+    'no-await-in-loop': 'off', // contrary to rule docs, awaited things often are NOT parallelizable
+    // permanently disable due to performance issues (using custom rule `@fluentui/max-len` instead)
+    'max-len': 'off',
+    // permanently disable due to perf problems and limited benefit
+    // see here for perf testing (note that you must run eslint directly)
+    // https://eslint.org/docs/developer-guide/working-with-rules#per-rule-performance
+    'no-empty-character-class': 'off',
+
+    /**
+     *
```

Comment:
I know this is a picky unimportant thing but all the blank lines starting the comments are bothering me, and IMO it would look nicer to get rid of them

Reply:
as you said, not relevant to this PR, we can address as follow up